### DISCUSSION TITLE
Missing some icons.

### DISCUSSION THREAD

**Discussion by RoshanDasan** [tailwindlabs/heroicons#1092] [Ideas]
When I review Heroicons for Icons, I find it to be more compact and well-structured. However, I've noticed that some important icons, such as those for popular social media platforms like Instagram, Facebook, Twitter, and GitHub, are missing.

**Comment by LarsFlieger** [comment]
Quite similar request here: https://github.com/tailwindlabs/heroicons/discussions/1088
